docker-compose.dev.yml

```diff
@@ -0,0 +1,70 @@
+version: "3.4"
+
+services:
+  backend:
+    container_name: backend
+    build:
+      context: ./backend
+      dockerfile: Dockerfile
+      target: dev
+    environment:
+      DATABASE_CONNECTION: "mongodb://mongodb-sharded:27017/cpe-kmutt-web"
+      DATABASE_USERAME: root
+      DATABASE_PASSWORD: CPE-KMUTT-DB
+      DATABASE_AUTH_SOURCE: admin
+      JWT_SECRET_KEY: very-secret-key.jwt
+      JWT_EXPIRES: 3600s
+      OPS_KEY: "CPE-KMUTT.opskey"
+      NODE_ENV: "development"
+      PORT: 3000
+    ports:
+      - 3000:3000
+    volumes:
+      - "./backend:/backend"
+    command: yarn start:dev
+    links:
+      - mongos
+      - shard-0
+      - mongodb-cfg
+    restart: always
+
+  mongos:
+    image: "bitnami/mongodb-sharded:4.2"
+    environment:
+      MONGODB_ADVERTISED_HOSTNAME: mongodb-sharded
+      MONGODB_SHARDING_MODE: mongos
+      MONGODB_CFG_PRIMARY_HOST: mongodb-cfg
+      MONGODB_CFG_REPLICA_SET_NAME: cfgreplicaset
+      MONGODB_REPLICA_SET_KEY: CPE-KMUTT-DB-CLUSTER
+      MONGODB_ROOT_PASSWORD: CPE-KMUTT-DB
+      TZ: Asia/Bangkok
+    ports:
+      - "27017:27017"
+    volumes:
+      - "./mongo/mongos:/bitnami"
```

Comment:
cannot mount from root folder don't know why
use `local volume` instead

Reply:
I suggest to use separate file `docker-compose.dev.win.yml` that contain only volume override 
and create script to run it
```bash
docker-compose -f docker-compose.yml -f docker-compose.dev.yml -f docker-compose.dev.win.yml up
```

Reply:
Example `docker-compose.dev.yml`
```yaml
services:
  mongos:
     volume:
       -  mongos:/bitnami

volumes:
  mongos:
```